src/services/stripeService.js

```
@@ -0,0 +1,286 @@
+
+    case "invoice.payment_failed":
+      newStatus = PAYMENT_STATUS.PAYMENT_FAILED;
+      break;
+
+    case "customer.subscription.deleted":
+      newStatus = PAYMENT_STATUS.CANCELED;
+      break;
+
+    case "customer.subscription.updated":
+      // Check if subscription is past_due or unpaid
+      const subStatus = event.data?.object?.status;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)